scripts/scil_split_tractograms.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+
+from dipy.io.stateful_tractogram import StatefulTractogram
+from dipy.io.streamline import save_tractogram
+import numpy as np
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist, add_reference_arg,)
+
+DESCRIPTION = """
+    Split a tractogram into multiple files, 2 options available :
+    Split into X files, or split into files of Y streamlines
+"""
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(
+        formatter_class=argparse.RawTextHelpFormatter,
+        description=DESCRIPTION)
+
+    p.add_argument('input_tractogram',
+                   help='Input filename to split (trk or tck)')
+    p.add_argument('output_name',
```

Comment:
In the old script, I should have mentioned that the extension is required. It should be added now